mm/2s2h/Enhancements/Cutscenes/MiscInteractions/SkipTatlInterrupts.cpp

```diff
@@ -38,7 +38,7 @@ void RegisterSkipTatlInterrupts() {
     // General interupt 2 (the flags were directly copied from the original code)
```

Comment:
Typo: `interupt` &#8594; `interrupt`

Reply:
These are issues upstream, so likely won't fix them here for now